lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -47,6 +48,7 @@
 from galaxy.workflow.reports import generate_report
 from galaxy.workflow.run import invoke, queue_invoke
 from galaxy.workflow.run_request import build_workflow_run_configs
+from galaxy.version import VERSION
```

Comment:
This should go between `from galaxy.util.sanitize_html import sanitize_html` and `from galaxy.web import (`